nautobot/extras/views.py

```diff
@@ -681,7 +681,7 @@ class ExportTemplateBulkDeleteView(generic.BulkDeleteView):
 
 class GitRepositoryListView(generic.ObjectListView):
     queryset = GitRepository.objects.all()
-    # filterset = filters.GitRepositoryFilterSet
+    filterset = filters.GitRepositoryFilterSet
```

Comment:
It's been that way since GitRepositories were added. IIRC it and the line below were commented out because we don't yet have a GitRepositoryFilterForm implementation, so at the time the filterset was thought to be unneeded. I think leaving the below line in place as a reminder of absent functionality is desirable.